lib/modules/platform/github/readme.md

```diff
@@ -2,13 +2,15 @@
 
 ## Authentication
 
-First, [create a classic Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic) for the bot account, select `repo` scope.
-Fine-grained Personal Access Tokens do not support the GitHub GraphQL API and cannot be used with Renovate.
+First, create a
+[fine-grained](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token)
+or [classic](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic)
+PAT ([Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#about-personal-access-tokens)) for the bot account, select `repo` scope.
 
 Let Renovate use your PAT by doing _one_ of the following:
 
 - Set your PAT as a `token` in your `config.js` file
-- Set your PAT as an environment variable `RENOVATE_TOKEN`
+- Set your PAT as an environment variable `RENOVATE_TOKEN` (generally via `${{ secrets.RENOVATE_TOKEN }}`)
```

Comment:
This is not correct, please revert

Reply:
If you're using a GitHub workflow, it would be, no?

Reply:
Err, in my case, I'm not using an env var at all, I'm using the `with:`/`token:`:
https://github.com/check-spelling-sandbox/renovatebot-test-wp-calypso/blob/fc66d304732ee4f7c7ab806d4b64427bfcc55b31/.github/workflows/renovate.yml#L92

```yml
      - uses: renovatebot/github-action@v39.0.1
        with:
          configurationFile: renovate.json5
          token: ${{ secrets.RENOVATE_TOKEN }}
```

Reply:
This is not the documentation repo for the GitHub Action either though

Reply:
Fair. Will drop